ambry-api/src/test/java/com.github.ambry/rest/MockBlobStorageService.java

```diff
@@ -109,6 +109,20 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
   }
 
   @Override
+  public void handlePut(RestRequest restRequest, RestResponseChannel restResponseChannel) {
+    if (shouldProceed(restRequest, restResponseChannel)) {
```

Comment:
ambry does not support PUTs. You should do the same thing you did in `AdminBlobStorageService` here.